_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -36,7 +37,7 @@ This rule applies to any `headers` attribute specified on a [`cell`][] within a
 
 ## Expectation 1
 
-Each target's [attribute value][] is a [set of space separated tokens][]. Each token is the value of the `id` attribute of an element, that is a [`cell`][] of the same [`table`][].
+Each target's [attribute value][] is a [set of space separated tokens][]. Each token is the value of the `id` attribute of an element, which is a [`cell`][] of the same [`table`][] and is a `th` element.
```

Comment:
I would feel much better about this if we had some test data showing that referencing `td`s, with and without role=row/columnheader doesn't work. Would it be possible to get someone to do some testing before we say this definitely fails WCAG?

Reply:
It is at least invalid HTML: [headers attribute](https://html.spec.whatwg.org/multipage/tables.html#attr-tdth-headers), and shouldn't create a relationship: [Forming relationship](https://html.spec.whatwg.org/multipage/tables.html#header-and-data-cell-semantics), step 7 only keeps the "headers".

Due to non-interference with the host language, I'd be surprised if that worked. But I agree we should test it nonetheless.

Reply:
I used this as a test with just regular HTML:
https://codesandbox.io/s/table-test-68d78y?file=/index.html

JAWS doesn't care if the referenced element is a `td` or a `th`, it reads the referenced headings in both cases. Surprisingly, NVDA and desktop Voiceover seem to ignore the `headers` attribute and use the automatic calculation - they won't read `td`'s, but they read the `th` at the top of the column rather than the `th` that I referenced via `headers`. 

So, 2 of the 3 desktop screen readers don't handle headers properly even when they're referencing a th. The 3rd doesn't care if it's a td or a th. Given that, I'm having a hard time seeing how this rule has any real world value - the results don't seem to be a factor of what the headers attribute is referencing.

Reply:
To be clear, I was using Chrome on Mac, and Edge on Windows.